src/components/shared/sidebar/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from "react";
+import { useMutation } from "react-query";
+import { HomeIcon, InboxIcon, PlusCircleIcon } from "@heroicons/react/24/outline";
+import Router from "next/router";
+
+import { logout } from "@/pages/api";
+import { useStore } from "@/pages/store/store";
+import NavItem from "./NavItem";
+
+export default function Sidebar() {
+  const { token, user } = useStore();
+  const [selected, setSelected] = useState("Menu");
+
+  const mutationLogout = useMutation(() => logout(token), {
+    onSuccess: () => {
+      useStore.getState().setToken("");
```

Comment:
so this line simplifies to this:
```suggestion
     setToken("");
```